src/context/concrete_base.rs

```diff
@@ -210,25 +214,45 @@ impl ConcreteBaseAudioContext {
     /// The sample rate (in sample-frames per second) at which the `AudioContext` handles audio.
     #[allow(clippy::cast_precision_loss)]
     #[must_use]
-    pub fn sample_rate(&self) -> f32 {
+    pub(super) fn sample_rate(&self) -> f32 {
         self.inner.sample_rate.0 as f32
     }
 
     /// The raw sample rate of the `AudioContext` (which has more precision than the float
     /// [`sample_rate()`](BaseAudioContext::sample_rate) value).
     #[must_use]
-    pub fn sample_rate_raw(&self) -> SampleRate {
+    pub(super) fn sample_rate_raw(&self) -> SampleRate {
         self.inner.sample_rate
     }
 
+    /// This represents the number of seconds of processing latency incurred by
+    /// the `AudioContext` passing the audio from the `AudioDestinationNode`
+    /// to the audio subsystem.
+    // We don't do any buffering between rendering the audio and sending
+    // it to the audio subsystem, so this value is zero. (see Gecko)
+    #[allow(clippy::unused_self)]
+    #[must_use]
+    pub(super) const fn base_latency(&self) -> f64 {
+        0.
+    }
```

Comment:
I followed what is done in Gecko / Firefox, but I'm not super sure it is correct so I though it was maybe better to have this API. Let me know what you think

Reply:
Yeah, I think this is about internal buffering, which we do not do. So probably 0 is fine

Reply:
Yup this is my impression too in our case, (it is just that there is something I don't really understand about the calculation they do in the spec with double buffering)